site2/docs/admin-api-namespaces.md

```diff
@@ -288,7 +288,7 @@ admin.namespaces().getNamespaceReplicationClusters(namespace)
 
 Backlog quota helps the broker to restrict bandwidth/storage of a namespace once it reaches a certain threshold limit. Admin can set the limit and take corresponding action after the limit is reached.
 
-  1.  producer_request_hold: broker holds but not persists produce request payload
+  1.  producer_request_hold: broker holds but not persists produce request payload until client configuration `operationTimeoutMs` is exceeded.
```

Comment:
This is not exactly correct. First, the broker does not hold the payload: it's the client that will keep retrying up to the `sendTimeout` configured in the producer.

Reply:
@merlimat  you are exactly right that broker actually does **not** hold the message. I will make an update.